src/pages/iou/request/step/IOURequestStepTaxAmountPage.tsx

```diff
@@ -150,12 +150,12 @@ function IOURequestStepTaxAmountPage({
             <MoneyRequestAmountForm
                 isEditing={Boolean(backTo || isEditing)}
                 currency={currency}
-                amount={transactionDetails?.taxAmount}
-                taxAmount={getTaxAmount(transaction, policy, Boolean(backTo || isEditing))}
+                amount={Math.abs(transactionDetails?.taxAmount ?? 0)}
```

Comment:
Note: This line specifically didn't come from https://github.com/Expensify/App/pull/42064, but it was included in this cherry-pick PR for some reason - it looks like a fix to something else that exists in `main` which is why I kept it

Reply:
Oh hey this change is from this PR - https://github.com/Expensify/App/pull/42039/files - which we also wanted to CP, so good we added